runtime/cudaq/platform/default/rest_server/RestServerMain.cpp

```diff
@@ -0,0 +1,221 @@
+/*******************************************************************************
```

Comment:
I think we may want to move this to the `tools` directory with a proper help message. If we end up creating extension points for both the client-QPU interaction (currently REST) as well as the execution logic (currently LLVM/MLIR JIT), then we really would have a generic QPU daemon, and I'd propose we name the tool `cudaq-qpud`.